web/src/features/moreCast2/forecastDraft.test.ts

```diff
@@ -0,0 +1,70 @@
+import { MorecastDraftForecast } from 'features/moreCast2/forecastDraft'
+import { DraftMorecast2Rows } from 'features/moreCast2/interfaces'
+import { buildValidActualRow, buildValidForecastRow } from 'features/moreCast2/rowFilters.test'
+import { DateTime, Settings } from 'luxon'
+
+// Temporarily set DateTime.now() to return the same DateTime when called
+const TEST_DATE = DateTime.fromISO('2024-01-01T00:00:00.000-08:00')
+Settings.now = () => TEST_DATE.toMillis()
```

Comment:
We can just mock `getDateTimeNowPST` to return `TEST_DATE` instead. I'm way of the scope of this setting change.

Reply:
Take a look at `web/src/features/auth/slices/authenticationSlice.test.ts` for how to mock with `sinon`